config/manifests/bases/backstage-operator.clusterserviceversion.yaml

```diff
@@ -52,4 +53,6 @@ spec:
   provider:
     name: Red Hat Inc.
     url: https://www.redhat.com/
-  version: 0.0.0
+  version: 0.2.0
```

Comment:
is this OK? setting 0.0.0 here is fine but we need a specific `skiprange` and `replaces` that changes with every release.

Reply:
I think this gets replaced by the `VERSION` defined in the `Makefile`. So it should be fine, I guess..